apps/crn-server/src/data-providers/research-outputs.data-provider.ts

```diff
@@ -95,7 +95,7 @@ export class ResearchOutputSquidexDataProvider
     take?: number;
     skip?: number;
     search?: string;
-    filter?: ResearchOutputFilter;
+    filter?: ResearchOutputFilter | string;
```

Comment:
as states previously you were trying to make your life easier by exposing filtering outside of data-provider. How would it work with Contentful? you definitely don't want to do that